onadata/libs/utils/logger_tools.py

```diff
@@ -240,6 +240,13 @@ def save_attachments(xform, instance, media_files):
                 mimetype=content_type,
                 name=filename,
                 extension=extension)
+        replaced_attachments = (
```

Comment:
I'm unclear on why the added code is inside the `f in media_files` loop, why is that necessary?

Reply:
not really. Removed that.